src/debug.c

```diff
@@ -8,6 +8,12 @@
 **  SPDX-License-Identifier: GPL-2.0-or-later
 */
 
+// set EXPORT_INLINE to 'extern inline' to force an instance of each
+// function marked with EXPORT_INLINE to be actually emitted, so that
+// it can e.g. be accessed from gdb or lldb, or from software using
+// libgap as a shared library
+#define EXPORT_INLINE extern inline
+
```

Comment:
This is one of the few changes which are not a result of "search and replace `static inline`".